vuestorefront/tests/test_product.py

```diff
@@ -0,0 +1,20 @@
+from ..schemas.product import get_product_list
+from . import common
+
+
+class TestProduct(common.TestVuestorefrontCommon):
+    def test_01_get_product_list_by_name(self):
+        # WHEN
+        res = get_product_list(
+            self.env, 1, 100, "", {}, name=self.product_tmpl_bin.name
+        )
+        # THEN
+        self.assertEqual(res[0], self.product_tmpl_bin)
```

Comment:
Maybe it would be good to add test when searching by barcode or default code, not only name?

Reply:
It would be good to add a lot more tests. Original repo has 0 tests:)